### PR TITLE
Third branch refactored code and added solution

### PR DESCRIPTION
**Pull request by RLudchenko**

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/index.jsp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: skill
+  Date: 4/25/2020
+  Time: 10:19 PM
+  To change this template use File | Settings | File Templates.
+--%>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/index.jsp]: let's remove comments created by idea

File: src/main/webapp/web.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <servlet>
+        <servlet-name>index</servlet-name>
+        <servlet-class>mate.academy.internetshop.controllers.IndexController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>index</servlet-name>
+        <url-pattern>/</url-pattern>
+    </servlet-mapping>
+</web-app>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/web.xml]: add empty line to the end of file

**Review by dmytro-hryhoruk — COMMENTED** [review]
let's set up travis ci to check your PR every time https://www.youtube.com/watch?v=BHSsWTbuow4

File: src/main/webapp/WEB-INF/views/index.jsp (hunk 1 above)
Comment (RLudchenko) [reply src/main/webapp/WEB-INF/views/index.jsp]: Done

File: src/main/webapp/web.xml (hunk 2 above)
Comment (RLudchenko) [reply src/main/webapp/web.xml]: Done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**